src/blocks/youtube.v1.yaml

```diff
@@ -1,9 +1,8 @@
 $schema: http://json-schema.org/draft-04/schema#
 title: YouTube video
 type: object
-properties:
+allOf:
   - $ref: ../misc/caption.v1.yaml
-  - $ref: ../misc/attribution.v1.yaml
```

Comment:
This was added in error. There has been no request for attribution for youtube videos.